TwilioVerifyDemo/NotificationExtension/NotificationService.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  NotificationService.swift
+//  TwilioVerifyDemo
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import UserNotifications
+import TwilioVerifySDK
+import TwilioVerifyDemoCache
+
+final class NotificationService: UNNotificationServiceExtension {
+
+  // MARK: - Properties
+  
+  var contentHandler: ((UNNotificationContent) -> Void)?
+  var bestAttemptContent: UNMutableNotificationContent?
+
+  private var accessGroup: String {
+    get {
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/6403871eb74f1d0e39b1793f8ce8165cd9cc4625/TwilioVerifyDemo/NotificationExtension/NotificationService.swift#L30"></span>Computed read-only properties should avoid using the get keyword.<br />
<code>implicit_getter</code> <code>NotificationService.swift:30</code></td>
    </tr>
  </tbody>
</table>